### DISCUSSION TITLE
Changing `inform_url` on adopted device

### DISCUSSION THREAD

**Discussion by emerconn** [Art-of-WiFi/UniFi-API-client#154] [Q&A]
Using this API, Is it possible to change the `inform_url` on a device that's already adopted?

**Comment by malle-pietje** [comment]
AFAIK this cannot be done through the API for a single device, only for all devices connected to a controller.

**Reply by emerconn** [reply]
This would work for my needs, though I am unsure how to achieve it.

**Reply by malle-pietje** [reply]
It's a matter of assigning a value to the **Console Hostname/IP** field in the Console/Controller settings and setting the field **Override inform host with the UniFi OS Console’s hostname/IP** to true.

These can be obtianed using `list_settings()`:
- Property override_inform_host in the section where key = super_mgmt
- Property hostname in the section where key = super_identity

Then update them with `set_super_mgmt_settings_base()` and `set_super_identity_settings_base()` respectively.
